pkgs/development/compilers/rust/cargo-auditable.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, fetchFromGitHub, makeRustPlatform, rustc, cargo, installShellFiles, stdenv }:
+{ lib, buildPackages, fetchFromGitHub, makeRustPlatform, installShellFiles, stdenv }:
```

Comment:
why are they reordered here?

Reply:
They are not reordered, I've added buildPackages and removed the now unused rustc and cargo.

Reply:
with the previous ordering it would be `fetchFromGitHub, makeRustPlatform, buildPackages`

Reply:
We don't really have an agreed order of nixpkgs guidlines. Hence I would not insist on it in reviews because it just adds noise.

Reply:
I don't really see a system in the previous ordering, it looks like historic grown